drink/src/chain_api.rs

```diff
@@ -180,10 +130,12 @@ mod tests {
         dest: AccountId32,
         value: u128,
     ) -> DispatchResultWithInfo<<RuntimeCall<MinimalRuntime> as Dispatchable>::PostInfo> {
-        let call = RuntimeCall::<MinimalRuntime>::Balances(
-            pallet_balances::Call::<MinimalRuntime>::transfer { dest, value },
-        );
-        sandbox.runtime_call(call, Some(MinimalRuntime::default_actor()).into())
+        sandbox.execute_with(|| {
+            RuntimeCall::<MinimalRuntime>::Balances(
+                pallet_balances::Call::<MinimalRuntime>::transfer { dest, value },
+            )
+            .dispatch(Some(MinimalRuntime::default_actor()).into())
+        })
```

Comment:
out of curiosity - why is this better?

Reply:
not sure why I did that, should be the same, I will rollback